### PR TITLE
[Haxe][Completion] Added autocomplete after 'from' and 'to'

### PR DESCRIPTION
**Pull request by SlavaRa**
autocompleteafterfromto

### CODE REVIEW

File: External/Plugins/ASCompletion/Completion/ASComplete.cs [hunk 1]
```diff
@@ -153,33 +153,7 @@ static public bool OnChar(ScintillaControl Sci, int Value, bool autoHide)
...
-                        if (expr != null) beforeBody = expr.ContextFunction == null || expr.BeforeBody;
-                        if (!beforeBody && features.codeKeywords.Contains(word)) return false;
-                        // override
-                        if (word == features.overrideKey) return ASGenerator.HandleGeneratorCompletion(Sci, autoHide, word);
-                        // import
-                        if (features.hasImports && (word == features.importKey || word == features.importKeyAlt))
-                            return HandleImportCompletion(Sci, "", autoHide);
-                        // public/internal/private/protected/static
-                        if (Array.IndexOf(features.accessKeywords, word) >= 0)
-                            return HandleDeclarationCompletion(Sci, "", autoHide);
```
Comment (SlavaRa) [inline External/Plugins/ASCompletion/Completion/ASComplete.cs:182]: lines 156-182 extracted to `private bool HandleWhiteSpaceCompletion(ScintillaControl sci, int position, bool autoHide)`

File: External/Plugins/HaXeContext/Completion/CodeComplete.cs [hunk 2]
```diff
@@ -48,5 +49,42 @@ public override bool IsStringInterpolationStyle(ScintillaControl sci, int positi
...
+            var positionFrom = sci.PositionFromLine(member.LineFrom);
+            for (var i = positionFrom; i < position; i++)
+            {
+                if (sci.PositionIsOnComment(position)) continue;
+                var c = (char)sci.CharAt(i);
+                if (c == '(' || c == '<') groupCount++;
+                else if (c == ')' || (c == '>' && sci.CharAt(i - 1) != '-')) groupCount--;
+                else if (c == '{' && groupCount == 0)
+                {
+                    if (i <= position) return false;
```
Comment (elsassph) [inline External/Plugins/HaXeContext/Completion/CodeComplete.cs]: This test is always true, isn't it?

Comment (SlavaRa) [reply External/Plugins/HaXeContext/Completion/CodeComplete.cs]: Nope, for example:
```csharp
yield return new TestCaseData("BeforeBody_2")
                    .SetName("class Some {|}")
                    .Returns(false);
yield return new TestCaseData("BeforeBody_4")
                    .SetName("class Some {}|")
                    .Returns(false);
```

Comment (elsassph) [reply External/Plugins/HaXeContext/Completion/CodeComplete.cs]: `return false` is correct, but I mean the loop is `for (var i = positionFrom; i < position; i++)` so `i` is always less than `position`

Comment (SlavaRa) [reply External/Plugins/HaXeContext/Completion/CodeComplete.cs]: Yes, but I do not fully understand, you see this as a mistake, or it is not immediately clear why this is done?

**Pull request merged**